src/main.rs

```diff
@@ -55,16 +57,12 @@ fn main() {
         ("init", Some(m)) => InitOptions::new(&m).exec(),
         ("build", Some(m)) => BuildOptions::new(&m).exec(),
         ("test", Some(m)) => TestOptions::new(&m, &config).exec(),
+        ("deploy", Some(m)) => DeployOptions::new(&m, &config).exec(),
```

Comment:
nit but it seems you're ordering these based on the expected usage path of these commands -- is that standard or should we adopt some convention like listing them in alphabetical order? I think this affects the ordering for `--help` and it seems most popular other CLI tools alphabetize

Reply:
this doesn't affect ordering
```
SUBCOMMANDS:
    build            Build services for the Oasis platform
    chain            Run a local Oasis blockchain
    clean            Remove build products
    config           View and edit configuration options
    deploy           Deploy services to the Oasis blockchain
    help             Prints this message or the help of the given subcommand(s)
    ifattach         Attach an interface definition json to a service.wasm
    ifextract        Obtain interface definition(s) from the requested location
    init             Create a new Oasis package
    set-toolchain    Set the Oasis toolchain version
    test             Run tests against a simulated Oasis runtime
```